modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -119,11 +119,8 @@ class Stats_Behavioural extends \NDB_Form
         $suproject_query = '';
         // Only search for subprojects if $currentProject is a positive int.
         if (is_numeric($currentProject) && intval($currentProject) >= 0) {
-            $subprojList ="";
             $subprojects = \Utility::getSubprojectsForProject($currentProject);
-            foreach ($subprojects as $key=>$val) {
-                $subprojList .= $key.",";
-            }
+            $subprojList = implode(',', array_keys($subprojects));
             $subprojList = substr($subprojList, 0, -1);
```

Comment:
Hey @johnsaigle Is this line supposed to be removed?
`$subprojList = substr($subprojList, 0, -1);`
reason I ask is because you removed it similar in the modules/statistics/php/stats_demographic.class.inc

Reply:
Yep, thanks!